src/Microsoft.AspNetCore.Server.Testing/xunit/SkipIfEnvironmentVariableNotEnabled.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Testing.xunit;
+
+namespace Microsoft.AspNetCore.Server.Testing
+{
+    /// <summary>
+    /// Skip test if a given environment variable is not enabled. To enable set environment variable 
+    /// to true for the test process.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
+    public class SkipIfEnvironmentVariableNotEnabledAttribute : Attribute, ITestCondition
```

Comment:
Started out that way, but the IIS attribute tends to tell more 'setup IIS' etc. Probably will add a property to the attribute to enable users to add more info.